### PR TITLE
Added home controller and root index.

### LINKED ISSUE #1: Deployment Environment
#### As a developer I want a continuous deployment environment setup for my new Rails application.

Acceptance criteria:
- [x] [GitHub](https://github.com/AppFolioOnboarding) is used for source
  control.
- [x] The project's version of both Ruby and Rails are used both locally and on
  Heroku.
- [x] The landing page, `/`, provides some content.
- [x] Every available controller action has corresponding tests.
- [x] [CircleCI](https://circleci.com/) is used for Continuous Integration.
- [x] Heroku is used to deploy the application.
- [x] Pull requests are automatically deployed as review applications.
- [x] The staging application automatically deploys whenever the master branch
  is updated.
- [x] The staging application automatically deploys only after all tests pass
  on CircleCI.
- [x] The staging application deployment can be promoted to the production
  application deployment.
- [x] [Pronto](https://github.com/mmozuras/pronto) is used to help with code
  style consistency.

Discussion Topics:
- [x] Where can we discover how long it takes to return an HTTP response?
- [x] When should we commit our code?

References:
* https://rvm.io/rvm/install#try-out-your-new-rvm-installation
* http://guides.rubyonrails.org/getting_started.html
* http://guides.rubyonrails.org/testing.html#functional-tests-for-your-controllers
* https://devcenter.heroku.com/articles/getting-started-with-rails4
* https://devcenter.heroku.com/articles/pipelines
* https://github.com/bbatsov/ruby-style-guide

### PR DESCRIPTION
**Pull request by wpliao1989**
Closes #1 

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [x] I have replaced `X` above with the associated issue number.
- [x] I have changed the issue's label to "pull request".
- [x] I have assigned the designated reviewer to this PR.
- [x] I have set the appropriate label on this PR.

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

**Comment by bboe** [comment]
@wpliao1989 I see the waiting on reviewer label, however, I wouldn't normally be notified that this pull request exists unless you also add me as an assignee. Can you do that?

File: app/assets/javascripts/home.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (bboe) [inline app/assets/javascripts/home.js]: Is this file necessary?

File: app/assets/stylesheets/home.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Home controller here.
```
Comment (bboe) [inline app/assets/stylesheets/home.scss]: Is this file necessary?

File: app/helpers/home_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```
Comment (bboe) [inline app/helpers/home_helper.rb]: Is this file necessary?

File: config/routes.rb [hunk 4]
```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  root 'home#index'
+  get 'home/index'
```
Comment (bboe) [inline config/routes.rb]: Is this route necessary?

File: test/controllers/home_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionDispatch::IntegrationTest
+  test 'should get index' do
+    get home_index_url
```
Comment (bboe) [inline test/controllers/home_controller_test.rb]: Great test!

**Comment by bboe** [comment]
This is a great start. A few comments that might require you to remove unnecessary files/statements.

Also, aside from the two commits to fix things on travis, all the other commits should be squashed together in order to represent an atomic unit -- e.g., the addition of a single feature. We'll often make "code style commits" in response to code review so that the history is additive, and then subsequently squash that commit into the appropriate places before merging.

 Are you familiar with the squashing process? If not, I can walk you through it.

**Comment by wpliao1989** [comment]
I haven't done any squashing, so I'll need your help.

**Comment by wpliao1989** [comment]
I removed the unnecessary files and made the commit messages look nicer. Check it out!

**Comment by bboe** [comment]
Great work!
